lib/request.js

```diff
@@ -168,11 +183,19 @@ Object.defineProperties(Request.prototype, {
   routeOptions: {
     get () {
       const context = this[kRouteContext]
-      const serverLimit = context.server.initialConfig.bodyLimit
       const routeLimit = context._parserOptions.limit
-      return {
-        bodyLimit: ((routeLimit || serverLimit))
+      const serverLimit = context.server.initialConfig.bodyLimit
+      const options = {
+        method: context.config.method,
+        url: this.raw.url,
```

Comment:
`url` do not comes from `route option`, it should be the same as `routerPath`

Reply:
should i remove it from route option or just it's value replace it with routerPath value

Reply:
Replace it with `routerPath` value is fine.

Reply:
done 👍